cli/src/pcluster/config/cluster_config.py

```diff
@@ -371,6 +371,7 @@ def __init__(
         deletion_policy: str = None,
         encryption_in_transit: bool = None,
         iam_authorization: bool = None,
+        accesspoint_id: str = None,
```

Comment:
this is a new customer facing feature, so a new entry in the CHANGELOG should be added.